source/_integrations/zha.markdown

```diff
@@ -73,15 +73,15 @@ In addition, it has support for "Zigbee groups" that enable native on-device gro
 
 ## Introduction
 
-ZHA {% term integration %} is a Zigbee gateway implementation that follows the standard Zigbee 3.0 specification (and earlier revisions). In Zigbee, there are three different device types: Zigbee Coordinator (ZC), Zigbee Router (ZR), and Zigbee End Device (ZED). A Zigbee network always has one (and no more) Zigbee Coordinator, however, a Zigbee network can have multiple Zigbee Routers and multiple Zigbee End Devices.
+This ZHA integration is a hardware independent Zigbee gateway implementation that can replace most proprietary Zigbee gateways/bridges/hubs/controllers. It supports pairing/joining devices that use the Zigbee 3.0 standard and earlier Zigbee standards for home automation and lighting. It follows the Zigbee PRO specification, a low-power wireless communication protocol using small digital radios to create a private Zigbee network mesh that can connect compatible devices within a local area.
```

Comment:
I don't think should mention "Zigbee 3.0", "earlier Zigbee standards", and "Zigbee PRO" directly in the introduction. We probably don't need to at all.